### PR TITLE
Auto-convert "greek small letter mu" to micro symbol

### LINKED ISSUE #46: "Units" parameter is not checked for valid units.
Passing "μV" fails, because μ is https://www.compart.com/de/unicode/U+03BC, but only µ (https://www.compart.com/de/unicode/U+00B5) is supported in BV according to the spec.

Also passing nonsense like "sV" results in an ungraceful error.

image

### PR DESCRIPTION
**Pull request by sappelhoff**
closes #46

### CODE REVIEW

File: .github/workflows/python_tests.yml [hunk 1]
```diff
@@ -5,6 +5,12 @@ on:
     branches: [ master ]
   pull_request:
     branches: [ master ]
+  create:
+    branches: [master]
+    tags: ['**']
+  schedule:
+    - cron: "0 4 1 * *"
```
Comment (sappelhoff) [inline .github/workflows/python_tests.yml:12]: runs the CI as a cron job every 1st day of a month at 4am in the night

Comment (hoechenberger) [reply .github/workflows/python_tests.yml:12]: Why not every day though?

Comment (sappelhoff) [reply .github/workflows/python_tests.yml:12]: I thought cause of the environment :grinning: , and we only need to cover the case where mne or numpy change something - that's not so frequent

Comment (sappelhoff) [reply .github/workflows/python_tests.yml:12]: are you happy with this PR otherwise?

**Review by hoechenberger — APPROVED** [review]

**Comment by sappelhoff** [comment]
Thanks for the review @hoechenberger!

**Pull request merged**